### ISSUE TITLE
[Firebase Notifier] Don't support 'Push Channel'

### ISSUE THREAD

**Issue by f1nder** [symfony/symfony-docs#16910]
### Symfony version(s) affected

6.1

### Description

The documentation 6.1 claims support for "Push channel". In fact, this is not so. Found only "Chat Channel" support.

### How to reproduce

```
namespace Symfony\Component\Notifier\Bridge\Firebase;


final class FirebaseTransport extends AbstractTransport
{   

 public function supports(MessageInterface $message): bool
    {
        return $message instanceof ChatMessage;
    }

```

**Comment by stof** [comment]
Documentation issues should be reported to https://github.com/symfony/symfony-docs/issues/

**Comment by mdoutreluingne** [comment]
Thanks for the reporting.

I agree, Firebase only supports "Chat Channel". I also did not find support for the "Push Channel".

There is a PR(#16249) to fix this but it still hasn't been merged.

**Comment by javiereguiluz** [comment]
Fixed in #16249. Thanks!
